enigma-coinjoin-client/src/index.js

```diff
@@ -56,7 +56,7 @@ class CoinjoinClient {
     }
 
     watch() {
-        this.ws.onmessage = (msg) => {
+        this.ws.on('message', (msg) => {
```

Comment:
Tests are timing out with the other syntax, I'm not sure if the issue is with 'onmessage`, `onopen` or both.